projects/plugins/jetpack/_inc/client/at-a-glance/index.jsx

```diff
@@ -178,25 +157,21 @@ class AtAGlance extends Component {
 				performanceCards.push( <DashBoost siteAdminUrl={ this.props.siteAdminUrl } /> );
 			}
 
-			if ( performanceCards.length ) {
```

Comment:
this check for lack of cards is now part of the rendering of the Section component.